zulip_bots/zulip_bots/finder.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import os
+from os.path import basename, splitext
+from typing import Any, Optional, Text, Tuple
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+
+def import_module_from_source(path: Text, name: Optional[Text]=None) -> Any:
```

Comment:
I think the second argument `name` should be of type `Text` and must be required. Now this method is always used with both passed arguments.

Reply:
Is there any reason that it must always be called with `name` argument ? 
If the `name`  is not specified, it is assigned [here](https://github.com/zulip/python-zulip-api/pull/401/files/7b7e10afdcc6b2822bb175e492ab88a99036625f#diff-7d6ba3f111cf7a3c9528de51bd6a1e1eR9).

Reply:
@SivagiriVisakan I meant that the current method's calls don't use the ability to pass only one argument. This method is always called with both arguments in production code. That's why I proposed to change type `name` to just `Text` because `if not name` is always `False` in production code. I agree that your current implementation of method is more flexible, but, unfortunately, we don't need this flexibility now.